adobe-dmeservices/user-sync.py
# finish sync summary

This PR finishes implementation of the sign summary story, adding new fields and tracking for sign only users, as well as simplifying the summary printout. Existing fields have been renamed or removed in favor of clearer definitions.

## Summary
* Add additional information for sign stats (sign only users, and other absent fields)
* Refactor summary printout code
* Definitions of fields now clearer


## Testing Steps

* Sign neptune tests have been corrected to pass in this PR, but should be addressed in a new issue to use the proper fixture/object instance approach.  In their current state, not quite right.
* No code/unit testing currently implemented since this spans many methods throughout the class.

![image](https://user-images.githubusercontent.com/37643772/99008782-705e5e00-250c-11eb-89b3-94b9bd9f12d8.png)





* Updated as a function of DMESVS 432 but does not depend on that.  Update is standalone.
